indico/modules/rb_new/client/js/common/rooms/modals.jsx

```diff
@@ -33,4 +34,9 @@ export default {
             {() => <RoomDetailsModal roomId={roomId} onClose={onClose} />}
         </RoomDetailsPreloader>
     ),
+    'room-edit': (onClose, roomId) => (
```

Comment:
I don't think we should handle this particular dialog here. We should probably keep all the admin stuff in local state, especially because we do not want to get the room data from redux for the admin area (might be outdated, also inactive rooms are never in the room list that's in redux) but rather send the ajax request to get the room data (during `componentDidMount`) from inside the component.